Lib/test/test_fileio.py

```diff
@@ -496,6 +496,19 @@ def testTruncate(self):
         self.assertEqual(f.seek(0, io.SEEK_END), 15)
         f.close()
 
+        f = io.open(TESTFN, 'r+b')
```

Comment:
`test_fileio` is specifically for `FileIO` tests, but you are not testing FileIO here, but BufferedIORandom. So your test should go into the relevant test class in `test_io`.

Reply:
Also, I think the test should exercise several different buffer sizes.

Reply:
Done for both. See if it is ok now.